### ISSUE TITLE
Error description isn't returned as array

### ISSUE THREAD

**Issue by pcarana** [NICMx/rdap-server#51]
According to [RFC 7483 section 6](https://tools.ietf.org/html/rfc7483#section-6):
> The basic structure of that response is an object class containing an
   error code number (corresponding to the HTTP response code) followed
   by a string named "title" and an array of strings named
   "description".

Currently the description is returned as a string, not as an array of strings.

Eg. the response to "/ip/..." is:
```javascript
{
	"rdapConformance": [
		"rdap_level_0"
	],
	"errorCode": "400",
	"description": "Invalid IPv4 address : ..."
}
```

**Comment by pcarana** [comment]
The renderized is now handled by [NICMx/rdap-renderer-api](https://github.com/NICMx/rdap-renderer-api), which is implemented in [NICMx/rdap-json-renderer](https://github.com/NICMx/rdap-json-renderer) and used by the server. So, the fix has been done at the JSON Renderer.

**Comment by TheRedTrainer** [comment]
Verified. The description is displayed as a string array:

{
	"rdapConformance": [
		"rdap_level_0"
	],
	"errorCode": "400",
	"description": [
		"Invalid IPv4 address : ..."
	]
}

**Comment by pcarana** [comment]
Closing, fixed in Red Dog v1.2
